src/parser/jobs/mnk/modules/Steppies.tsx

```diff
@@ -0,0 +1,122 @@
+import {Plural, Trans} from '@lingui/react'
+import React from 'react'
+
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+
+import {CastEvent, DamageEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import Combatants from 'parser/core/modules/Combatants'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+const LEAD_BOOT_POTENCY = 300
+
+// Typically a player with lag can derp positional in opener, this usually happens 2 times per fight.
```

Comment:
I don't know shit about optimal MNK play so feel free to ignore this one, but if 1-2 misses can happen due to lag, these severities strike me as a bit, well, severe.

Reply:
From discord but, tl;dr gonna bump majors a bit since it's a very easy issue for users to fix and a _big_ DPS loss, but leaving mediums alone since even with lag you can just pre-pull True North or Riddle of Earth. 5.0 fights have enough raid damage to make RoE easy to use, plus TN has charges now.